lib/internal/filter.js

```diff
@@ -1,31 +1,19 @@
-import arrayMap from 'lodash/_arrayMap';
-import property from 'lodash/_baseProperty';
+import filter from 'lodash/_arrayFilter';
 import noop from 'lodash/noop';
 import once from './once';
 
 export default function _filter(eachfn, arr, iteratee, callback) {
     callback = once(callback || noop);
-    var results = [];
+    var truthy = [];
```

Comment:
Can we change this variable to to something like `var truthyIndexes` or `filteredIndexes`.

I'd also be curious about the performance of making an array of accepted indexes rather than a truthy list of indexes:

For instance an array might look like `[1, 5, 6]` if we wanted to take those indexes

the final filter array building step would look like

```js
var length = filteredIndexes.length;
var results = new Array(length)
for (var i = 0; i < length; i++) {
   results[i] = arr[filteredIndexes[i]];
}
return results
```

I'd expect this to be faster